templates/aws-stack.yml

```diff
@@ -2,15 +2,17 @@
 AWSTemplateFormatVersion: "2010-09-09"
 Description: "Buildkite stack %v"
 
+Transform: AWS::Serverless-2016-10-31
+
 Metadata:
   AWS::CloudFormation::Interface:
     ParameterGroups:
       - Label:
           default: Buildkite Configuration
         Parameters:
-        - BuildkiteAgentToken
         - BuildkiteAgentTokenParameterStorePath
         - BuildkiteAgentTokenParameterStoreKMSKey
+        - BuildkiteAgentToken
```

Comment:
This is deliberately moved to give lower precedence to the inline token option vs using an SSM Parameter